.github/workflows/check_build.yml

```diff
@@ -0,0 +1,21 @@
+name: Build EE2 Docker Image
+
+on:
+  pull_request
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      -
+        name: Build and push
+        uses: docker/build-push-action@v6
+        with:
+          push: false
+          tags: ee2/test:test
```

Comment:
1. Is QEMU setup necessary?
2. format each step with spacing to make it easier to read.

Reply:
I copied this directly from https://github.com/docker/build-push-action

Reply:
👍